src/Sfx/App/Scripts/Directives/RingViewDirective.ts

```diff
@@ -0,0 +1,231 @@
+module Sfx {
+
+  
+
+  export class RingViewDirective implements ng.IDirective {
+    public restrict = "AE";
+    public replace = true;
+    public templateUrl = "partials/ring-view.html";
+    public scope = {
+        nodes: "=",
+        clusterManifest: "=",
+    };
+
+    public link($scope: any, element: JQuery, attributes: any, ctrl: DetailViewPartController) {
+      let makeTippy = function(node, html){
```

Comment:
can we type these inputs and make it an arrow operator. (node, html) => instead of function(node, html)